class.jetpack.php

```diff
@@ -3786,12 +3786,20 @@ function admin_page_load() {
 					$error = $registered->get_error_code();
 					Jetpack::state( 'error', $error );
 					Jetpack::state( 'error', $registered->get_error_message() );
+					JetpackTracking::record_user_event( 'jpc_register_fail', array(
+						'error_code' => $error,
+						'error_message' => $registered->get_error_message()
+					) );
```

Comment:
Since we're just breaking after this, and not returning, is it possible to track a failure here, and then track a success on line `3799`?